src/main/frontend/components/NavBar.js

```diff
@@ -59,6 +60,7 @@ const NavBar = props => {
         <Route exact path="/genres/:id" component={GenreShow} />
         <Route exact path="/films/new" component={FilmForm} />
         <Route exact path="/films/:id" component={FilmShow} />
+        <Route exact path="/films/:id/delete" component={DeleteFilm} />
```

Comment:
Is there currently a `<Link>` to this delete page for a user to use to navigate there?  Also it looks like there is a delete button on a films show page, but also a delete page for similar functionality.  Is the goal to have both pages?  If so you can break out the delete fetch into it's own file and then import that in both components.